### PR TITLE
fix bug in http2 config names

### PR DESCRIPTION
**Pull request by pjfanning**
see #6 and #5

### CODE REVIEW

**Review by nvollmar — APPROVED** [review]
lgtm

File: src/main/g8/src/main/scala/com/example/helloworld/GreeterServer.scala [hunk 1]
```diff
@@ -37,7 +37,7 @@ object GreeterServer {
 
   def main(args: Array[String]): Unit = {
     // important to enable HTTP/2 in ActorSystem's config
-    val conf = ConfigFactory.parseString("pekko.http.server.enable-http2 = on")
+    val conf = ConfigFactory.parseString("pekko.http.server.preview.enable-http2 = on")
```
Comment (He-Pin) [inline src/main/g8/src/main/scala/com/example/helloworld/GreeterServer.scala:40]: Can we change the config so it works in new version?

Comment (pjfanning) [reply src/main/g8/src/main/scala/com/example/helloworld/GreeterServer.scala:40]: we need the quickstart to work with the current version - we can't just wait for a future release

**Review by He-Pin — APPROVED** [review]
Lgtm

**Pull request merged**